packages/platform-ios/native_modules.rb

```diff
@@ -5,24 +5,22 @@
 #
 require 'pathname'
 
-def use_native_modules!(root = "..", config = nil)
+def use_native_modules!(config = nil)
   if (!config)
     json = []
 
-    # Make sure `react-native config` is ran from your project root
-    Dir.chdir(root) do
-      IO.popen("./node_modules/.bin/react-native config") do |data|
-        while line = data.gets
-          json << line
-        end
+    IO.popen("npx react-native config") do |data|
+      while line = data.gets
+        json << line
       end
     end
 
     config = JSON.parse(json.join("\n"))
   end
 
+  project_root = Pathname.new(config["project"]["ios"]["sourceDir"])
+
   packages = config["dependencies"]
-  config_root = config["root"]
```

Comment:
can you make sure this doesn't break setups like Slider? https://github.com/react-native-community/react-native-slider/blob/master/react-native.config.js

Reply:
I guess I need more context on this setup because the configuration looks a bit unfamiliar to me. 

@michalchudziak

Reply:
Ah, I see. Looks like we shouldn't break it because we are only reading `config.root` and `project`, not a `dependency` namespace, but will save it down to further debug.